packages/components/src/core/Button/index.stories.tsx

```diff
@@ -56,121 +62,210 @@ export const Default = {
+  display: "contents",
+};
+const stateLevel = {
+  marginBottom: 10,
 };
 
-const minimalPlacementStyles = {
-  display: "grid",
-  gridColumnGap: "24px",
-  gridRowGap: "0px",
-  gridTemplateColumns: "repeat(3, min-content)",
-  gridTemplateRows: "1fr",
+const styleLabel = {
+  fontSize: "2em",
+  fontWeight: "normal",
+  gridColumn: "2 / 6",
+  marginBottom: 0,
+};
+const typeLabel = {
+  borderStyle: "solid none none none",
+  borderWidth: "2px",
+  fontSize: "1.17em",
+  fontWeight: "normal",
+  gridColumn: "2 / 6",
+  justifySelf: "stretch",
+  margin: "20px 0",
+  paddingTop: 10,
+};
+const iconLabel = {
+  alignSelf: "end",
+  borderStyle: "solid none none none",
+  borderWidth: "1px",
+  fontWeight: "normal",
+  gridColumn: "2 / 6",
+  justifySelf: "stretch",
+  margin: "0 0 5px 0",
+  paddingTop: 10,
+};
+const disabledLabel = {
+  alignSelf: "end",
+  fontWeight: "normal",
+  gridColumn: "1 / 2",
+  marginTop: 0,
+};
+const stateLabel = {
+  fontWeight: "normal",
+  margin: "10px 0",
 };
 
-export const MinimalLivePreview = {
+export const LivePreview = {
   parameters: {
     snapshot: {
       skip: true,
     },
   },
   render: (props: Args): JSX.Element => {
+    // loop through all SDS_STYLES
     return (
-      <div style={minimalPlacementStyles as React.CSSProperties}>
-        <RawButton {...props} sdsStyle="minimal" sdsType="primary">
-          {text}
-        </RawButton>
-
-        <RawButton
-          {...props}
-          startIcon={<Icon sdsIcon="download" sdsSize="s" sdsType="button" />}
-          sdsStyle="minimal"
-          sdsType="primary"
-        >
-          {text}
-        </RawButton>
-        <RawButton {...props} sdsStyle="minimal" sdsType="secondary">
-          {text}
-        </RawButton>
-      </div>
+      <>
+        {SDS_STYLES.map((sdsStyle) => {
+          return <ButtonStyleOption sdsStyle={sdsStyle} key={sdsStyle} />;
+        })}
+      </>
     );
+
+    // loop through all SDS_TYPES + create headers for SDS_STYLES
+    function ButtonStyleOption({
+      sdsStyle,
+    }: {
+      sdsStyle: (typeof SDS_STYLES)[number];
+    }) {
+      return (
+        <div style={styleLevel}>
+          <h3 style={styleLabel}>
+            Style: <b>{sdsStyle}</b>
+          </h3>
+          {SDS_TYPES.map((type) => {
+            return (
+              <ButtonTypeOption sdsStyle={sdsStyle} type={type} key={type} />
+            );
+          })}
+        </div>
+      );
+    }
+
+    // loop through all ICON_OPTIONS + create headers for SDS_TYPES
+    function ButtonTypeOption({
+      sdsStyle,
+      type,
+    }: {
+      sdsStyle: (typeof SDS_STYLES)[number];
+      type: (typeof SDS_TYPES)[number];
+    }) {
+      return (
+        <div style={displayContents}>
+          <h4 style={typeLabel}>
+            Type: <b>{type}</b>
+          </h4>
+          {/* Minimal Secondary doesn't have icon button option */}
+          {sdsStyle === "minimal" && type === "secondary" ? (
```

Comment:
@liaprins this is the only change that affects the screenshot. I added secondary icon button examples for `rounded` and `square`, since your comment said only `minimal` style doesn't have secondary icon button. PTAL thank you!

Reply:
Ah good catch, thank you!